optaweb-employee-rostering-frontend/src/store/store.ts

```diff
@@ -77,5 +78,6 @@ export function configureStore(
   );
 
   restServiceClient.setDispatch(store.dispatch);
+  restServiceClient.setStore(store);
```

Comment:
Circular dependency. REST client needs store (to check connection status - I think it shouldn't) and a REST client instance is needed to create the store. I would classify this as a code smell.